README.rst

```diff
@@ -80,6 +80,11 @@ Enables and configures the postgresql plugin. Needs refinement.
 
 Enables and configures the syslog plugin.
 
+``collectd.write_graphite``
+------------
```

Comment:
This needs to cover the whole of previous line's text. Something like the following:
```
``collectd.write_graphite``
---------------------------------
```